src/fixtures/themes.js

```diff
@@ -81,6 +81,14 @@ export const Themes = (arch) => {
         htmlFor: 'Password',
         color: isCheri ? '#000' : '#fff',
       },
+      warning: {
+        fontFamily: isCheri ? '000' : 'Monaco',
+        fontStyle: 'normal',
+        fontWeight: '400',
+        fontSize: isCheri ? '16px' : '20px',
+        lineHeight: '32px',
```

Comment:
Is it me or is `lineHeight` a bit big here. Maybe consider decreasing line height a little and adding a top margin